conan/__init__.py

```diff
@@ -1,6 +1,6 @@
 from conans.model.conan_file import ConanFile
-from conan.tools.scm import Version
+from conan.tools.scm import Version as _Version
```

Comment:
This is necessary to not accidentally expose ``from conan import Version`` accidentally